libdrgn/program.c

```diff
@@ -433,14 +433,13 @@ drgn_program_set_core_dump(struct drgn_program *prog, const char *path)
 		prog->file_segments[j].file_size = phdr->p_filesz;
 		prog->file_segments[j].fd = prog->core_fd;
 		prog->file_segments[j].eio_is_fault = false;
+		/*
+		 * Only zerofill between p_filesz and p_memsz for saved
+		 * kernel cores. makecoredump excludes zero pages; userland
```

Comment:
I missed this earlier, but the tool is called `makedumpfile`. The same thing needs to be fixed in the commit message.

```suggestion
		 * kernel cores. makedumpfile excludes zero pages; userland
```